### ISSUE TITLE
Accessibility Insights Fastpass causes pages using the monaco editor to crash

### ISSUE THREAD

**Issue by mryanlo** [microsoft/monaco-editor#2546]
Repro steps:
1. Open a page in some website/app that uses the monaco editor. For example: https://microsoft.github.io/monaco-editor/
2. Install the [Microsoft Accessibility Insights chrome extension](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni?hl=en-US). 
3. Run the Fast Pass check on the page from step 1.
![image](https://user-images.githubusercontent.com/55516393/123315005-154c0b00-d4e0-11eb-9dd2-f7ee3a893ad8.png)
4. Observe the fast pass takes longer to run and after a few seconds, the page crashes with an out of memory error:
![image](https://user-images.githubusercontent.com/55516393/123315101-2e54bc00-d4e0-11eb-979e-7bd69bbc5dde.png)

Expected: fast pass should be able to run on all parts of the page to test for accessibility issues even if the page has a monaco editor in it.
Actual: the out of memory error

We need to run this fast pass command on our app's pages to make sure they are accessibility compliant for our customers. We can't do so if the monaco editor is causing some to crash, and we're planning on adding the monaco editor to more parts of our app soon.

Browser: Google Chrome (Microsoft Edge works okay, but the fast pass will run very slowly)

**Comment by alexdima** [comment]
I suggest that you please report this issue to the `Accessibility Insights for Web` Chrome extension.

**Comment by mryanlo** [comment]
@alexdima thanks for your response. I just wanted to ask before doing so that we can confirm this is not an issue with the microsoft monaco editor?

**Comment by alexdima** [comment]
@mryanlo I don't believe so. The editor does not cause allocations on the JS heap, so it cannot lead to OOM exceptions, unless typing in the editor, running a command in the editor, etc. Otherwise, when idle, the editor doesn't allocate barely anything.
